tests/settings.py

```diff
@@ -98,8 +106,11 @@
     database_config["postgres"]["HOST"] = "127.0.0.1"
     database_config["postgres"]["PORT"] = "5432"
```

Comment:
You should set the same parameters also for the replica database.
Could you rename `postgres_2` to `postgres_replica` please?

Reply:
Done.